src/pages/StaticPages/Ferpa.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import VerticalNavigation from "../../shared/VerticalNavigation";
+import { RootState } from "../../app/common/store";
+import { requestModulesAndProblems } from "../CodeEditor/CodeEditorSlice";
+import { useLocation } from "react-router-dom";
+import { adminPathRegex } from "../../shared/constants";
+import { styled } from '@mui/styles'
+import theme from "../../shared/theme";
+import { Typography } from '@mui/material';
+import Footer from '../../shared/Footer';
+
+interface ProblemLocationState {
+    moduleName?: string;
+}
+
+const TitleHolder = styled("div")({
+  height: 400,
+  width: "100%",
+  backgroundColor: "white",
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "center",
+  alignItems: "center",
+  background: 'linear-gradient(transparent, transparent, transparent, transparent, #f1f1f11a, #f1f1f129, #f1f1f1da), url(https://www.transparenttextures.com/patterns/inspiration-geometry.png)',
+  backgroundImage: 'url(https://www.transparenttextures.com/patterns/inspiration-geometry.png)',
+  backgroundSize: "35em",
+  mozWindowShadow: 'inset 0 -10px 10px -10px #0000001a',
+  webkitBoxShadow: 'inset 0 -10px 10px -10px #0000001a',
+  boxShadow: 'inset 0 -10px 10px -10px #0000001a',
+});
+
+const TextHolder = styled('div')({
+    width: '70%',
+    maxWidth: "1400px",
+    padding: 30,
+    textAlign: 'left',
+    backgroundColor: 'white',
+    [theme.breakpoints.down("md")]: {
+        width: '100%'
+      },
+})
+
+function Ferpa() {
+    const modules = useSelector((state: RootState) => {
+        const sortedModules = state.codeEditor.navStructure;
+        return sortedModules.map((value) => value.name);
+    });
+    const dispatch = useDispatch();
+    const locationState = useLocation<ProblemLocationState>();
+
+    useEffect(() => {
+        dispatch(
+          requestModulesAndProblems({
+            isAdmin: adminPathRegex.test(locationState.pathname),
+          })
+        );
+        //disable exhaustive dependencies
+        //eslint-disable-next-line
+    }, [dispatch]);
+
+  return (
+    <>
+    <div id="top"/>
+    <VerticalNavigation
+        light={true}
+        modules={modules}
+        codingPage={false}
+    />
+    <TitleHolder>
+        <div>
+            <Typography variant="h2">
+              FERPA Addendum - Edugator
+            </Typography>
+        </div>
+    </TitleHolder>
+    <div style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: '#fcfcfc'
+    }}>
+        <TextHolder>
+            <Typography variant="body1" sx={{lineHeight: 2}}>
+                <b>What is FERPA?</b>
+                <br/>
+                <br/>
+                FERPA, the Family Education Rights and Privacy Act, is an American federal law that governs how institutions protect student Education Records.
```

Comment:
Do you remember @digitdustin if the process of converting static pages was manual or did you use a script or a third party to convert Word docs into typescript? I remember I gave you Word docs? You might want to document this for future references.

Reply:
This was all manual, can document on Notion